spinnaker/clouddriver
# feat(test): extend ECS create server group tests for artifacts

This PR contains integration tests for Create Server Group Operation for the task definition type as artifacts.

There are total 2 tests:

- Create server group operation for the task definition type as artifacts, launch type as EC2 with target group mappings.

- Create server group operation for the task definition type as artifacts, launch type as FARGATE target group mappings.

This change also adds further checks for 2 tests mentioned above, including:

- checking the task status after creation to ensure the ECS Service has been created

- ensures the expected ECS services calls were made

- ensures the content of the task definition and service reflect expected values given test input.
